stdlib/kconv/0/kconv.rbs

```diff
@@ -0,0 +1,166 @@
+# <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+# Kanji Converter for Ruby.
+#
+module Kconv
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # ASCII
+  #
+  ASCII: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # Auto-Detect
+  #
+  AUTO: nil
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # BINARY
+  #
+  BINARY: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # EUC-JP
+  #
+  EUC: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # ISO-2022-JP
+  #
+  JIS: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # NOCONV
+  #
+  NOCONV: nil
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # Shift_JIS
+  #
+  SJIS: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # UNKNOWN
+  #
+  UNKNOWN: nil
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # UTF-16
+  #
+  UTF16: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # UTF-32
+  #
+  UTF32: Encoding
+
+  # <!-- rdoc-file=ext/nkf/lib/kconv.rb -->
+  # UTF-8
+  #
+  UTF8: Encoding
+
+  # <!--
+  #   rdoc-file=ext/nkf/lib/kconv.rb
+  #   - Kconv.guess(str)   => encoding
+  # -->
+  # Guess input encoding by NKF.guess
+  #
+  def self.guess: (String str) -> Encoding?
```

Comment:
I couldn't find any cases where `::Kconv.guess` returns `nil`. Wouldn't `Encoding` be sufficient?

Reply:
Sorry, I'm not sure. Japanese Ruby Reference manual says it may return nil.
https://docs.ruby-lang.org/ja/latest/class/Kconv.html#M_GUESS

But the English version says it always returns an Encoding object.
https://docs.ruby-lang.org/en/3.3/Kconv.html#method-i-guess

I'll take a look the implementation later.

Reply:
Hmm... it seems `Kconv.guess` always returns an Eencoding object, right? I'll fix the definition and the documentation tomorrow.

https://github.com/ruby/nkf/blob/master/lib/kconv.rb#L141
https://github.com/ruby/nkf/blob/master/ext/nkf/nkf.c#L191

Thank you for reviewing!

Reply:
Fixed.